app/views/catalog/_report_biased_results_form.html.erb

```diff
@@ -7,7 +7,7 @@
     You can help us address this issue by reporting any biased or harmful results you encounter on Library search tools by using the form below. If you would prefer to report this anonomously, you may leave the name end email fields blank.
     <br/>
     <br/>
-    You are reporting bias in the following <%= link_to 'search results', form.context %>
+    You are reporting bias in the following <%= link_to 'search results', form.context, target: '_blank' %>
```

Comment:
Could we please also include an indication to screen reader users that this opens in a new tab?  Either in the link text itself, or we could add a little icon like in #3634?

Reply:
Great idea!  I'll do this after this morning's demo.